fixtures/js-stubs/opsgenieIntegration.tsx

```diff
@@ -0,0 +1,30 @@
+import {OrganizationIntegration} from 'sentry/types';
+
+export function OpsgenieIntegration(
+  params: Partial<OrganizationIntegration> = {}
+): OrganizationIntegration {
+  return {
+    id: '1',
+    name: 'hello-world',
+    icon: null,
+    domainName: 'hello-world.app.opsgenie.com',
+    accountType: null,
+    status: 'active',
+    provider: {
+      key: 'opsgenie',
+      slug: 'opsgenie',
+      name: 'Opsgenie (Integration)',
```

Comment:
Once we G/A, do we have a plan to remove the `(Integration)` text?

Reply:
I can remove it now, since I don't think the name is read anywhere.